### DISCUSSION TITLE
error  Replace `typeof·componentSizes` with `(typeof·componentSizes)`  prettier/prettier

### DISCUSSION THREAD

**Discussion by lixiaofa** [prettier/prettier#14942] [Q&A]
`
export type ComponentSize = typeof componentSizes[number]
`
![image](https://github.com/prettier/prettier/assets/23230215/f48ad3ae-64f8-41a2-96db-85831f6cfa88)

**Comment by kachkaev** [comment]
👋 @lixiaofa! Sorry, but it is not clear how your issue is related to Prettier Core. I guess you use [eslint-plugin-prettier](https://github.com/prettier/eslint-plugin-prettier) in the codebase, so if you just follow the suggestions, CI will pass:


```diff
-export type ComponentSize = typeof componentSizes[number]
+export type ComponentSize = (typeof componentSizes)[number] 
```
